### DISCUSSION TITLE
Extra dependecies

### DISCUSSION THREAD

**Discussion by baltsevicha** [wix/react-native-navigation#7399] [Enhancements & New Ideas]
## 🚀 Feature

It would be cool to remove all state management libs from dependencies (like mobx)

- ✅ <strong>Have you read the [Contributing Guidelines on issues](https://wix.github.io/react-native-navigation/docs/meta-contributing)?</strong>
Yes

## Motivation

We use Redux in our project and we don't use mobx in our project, but We have some warnings that we need to change React version as mobx-react requires other versions in peer dependencies. It's not cool

## Pitch

I don't think a navigation library should include dependencies like "mobx" or "lodash". I understand that it might be quite difficult to rewrite the library without using popular solutions. But having 2 different state management libraries in the same project sounds even weirder. Your library is downloaded over 20,000 times a week. I think if you make such an improvement, it will save a lot of resources in the aggregate.

### Are you willing to resolve this issue by submitting a Pull Request?

 - ✖️  &nbsp; No, I don’t have the time and I’m okay to wait for the community / maintainers to resolve this issue.

**Comment by swabbass** [comment]
@yogevbd
